core/src/sql/sql_provider_datafusion/mod.rs

```diff
@@ -229,23 +229,14 @@ impl<T, P> Display for SqlTable<T, P> {
     }
 }
 
-static ONE_COLUMN_SCHEMA: LazyLock<SchemaRef> =
-    LazyLock::new(|| Arc::new(Schema::new(vec![Field::new("1", DataType::Int64, true)])));
 
 pub fn project_schema_safe(
     schema: &SchemaRef,
     projection: Option<&Vec<usize>>,
 ) -> DataFusionResult<SchemaRef> {
     let schema = match projection {
         Some(columns) => {
-            if columns.is_empty() {
```

Comment:
Hmm, the reason this was added is because its needed for the unparser, so this change will break the federated table case.

Reply:
> Hmm, the reason this was added is because its needed for the unparser, so this change will break the federated table case.

for example? I try fix them.

Reply:
It is the same SQL, but its when its used with the `datafusion-federation` crate.